### PR TITLE
theme designer: switch demo component to dropdown

### PR DESCRIPTION
**Pull request by emmayjiang**
Switches the dropdown in the demo to the unstable dropdown. The change is also reflected in code sandbox.

image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/ee025ccd3a919f2f4dcdd3a74dbe6d2abfed24ff'>ee025ccd3a919f2f4dcdd3a74dbe6d2abfed24ff</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/Avatar-mount_base.html">1459</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/Avatar-mount.html">1513</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/Button-mount_base.html">918</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/Button-mount.html">958</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/FluentProvider-mount_base.html">1683</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/FluentProvider-mount.html">1675</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/FluentProviderWithTheme-mount_base.html">565</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/FluentProviderWithTheme-mount.html">567</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">568</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">564</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">580</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/MakeStyles-mount_base.html">1930</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/MakeStyles-mount.html">1915</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/SpinButton-mount_base.html">2945</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24090/perf-test-react-components/SpinButton-mount.html">3015</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Review by smhigley — COMMENTED** [review]
LGTM! But might I suggest we actually use `Select`? It should fit this use case, and the native control has better accessibility and usability across different platforms 😊

File: packages/react-components/theme-designer/src/components/Demo/Demo.tsx [hunk 1]
```diff
@@ -104,46 +102,27 @@ export const Column1 = () => {
...
-          </MenuItemCheckbox>
-        </MenuList>
-      </MenuPopover>
-    </Menu>
-  );
-};
-
 export const Column2 = () => {
   const styles = useStyles();
+  const dropdownId = useId('dropdown-default');
```
Comment (smhigley) [inline packages/react-components/theme-designer/src/components/Demo/Demo.tsx:125]: I don't actually see where this is applied to a label as an `id`

**Review by smhigley — APPROVED** [review]
Actually, one other thing 😅

**Pull request merged**